src/Sylius/Bundle/AdminBundle/MessageHandler/CreateAdminUserHandler.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\MessageHandler;
+
+use Sylius\Bundle\AdminBundle\Exception\CreateAdminUserFailedException;
+use Sylius\Bundle\AdminBundle\Message\CreateAdminUser;
+use Sylius\Component\Core\Model\AdminUserInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\User\Canonicalizer\CanonicalizerInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+use Symfony\Component\Validator\ConstraintViolationListInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+final class CreateAdminUserHandler implements MessageHandlerInterface
+{
+    private const GROUPS = ['sylius', 'sylius_user_create'];
```

Comment:
Extract these groups into parameters so that they are not hardcoded or maybe use the existing parameter 🤔